projects/dxc-ngx-cdk/src/lib/dxc-file-input/dxc-file-input.component.ts

```diff
@@ -871,6 +871,7 @@ export class DxcFileInputComponent
     }
     this.fileService.delete(this.requests.removeRequest.url, fileData).pipe(take(1)).subscribe((response) => {
       const arr: FileData[] = [];
+      this.onChangeRegister({ files: arr, eventType: EventType.REMOVE });
```

Comment:
eventType: EventType.REMOVE  this is not required as we are able to check the previous value of control and new value and compare it. hence remove this and change it accordingly.

Reply:
value change subscription at consuming service is getting called again and again until complete remove event is not succss so we need some identifier to understand that the remove event is emitted

Reply:
Please check with whom changes its calling again and again. for right now i am merging the code. Please check and fixed it